util/throttle.cc

```diff
@@ -30,6 +30,7 @@
 #include "util/cache2.h"
 #include "util/db_list.h"
 #include "util/flexcache.h"
+#include "util/hot_threads.h"
 #include "util/throttle.h"
 
 namespace leveldb {
```

Comment:
We have a minor race condition where if someone calls ThrottleShutdown() between the check of gThrottleRunning on line 94 and the locking of gThrottleMutex on line 107, the caller of ThrottleShutdown() will block for a full minute waiting to join the ThrottleThread; this can be eliminated by checking gThrottleRunning after locking gThrottleMutex on line 107.